reactular.test.tsx

```diff
@@ -14,6 +14,8 @@ import * as React from 'react';
 import { Simulate } from 'react-dom/test-utils';
 import { reactular } from './reactular';
 
+const wait = () => new Promise(resolve => setTimeout(resolve, 10));
```

Comment:
This is an alternative to using `act` which I couldn't use because Jest thinks we're not in a React environment.